src/Application/Matching/Adapter.php

```diff
@@ -71,7 +71,7 @@ private function addAmountWithoutSavingFundingsToDB(CampaignFunding $funding, st
          * @psalm-suppress PossiblyInvalidArrayAccess
          * @psalm-suppress PossiblyFalseReference - we know incrBy will retrun an array in multi mode
          */
-        [$initResponse, $fundBalanceFractional] = $this->storage->multi()
+        [$_, $fundBalanceFractional] = $this->storage->multi()
```

Comment:
Possibly more helpful for future editors to use underscore but with a more descriptive name still?

Reply:
```suggestion
        [$_initResponse, $fundBalanceFractional] = $this->storage->multi()
```